### DISCUSSION TITLE
Meal planner printing

### DISCUSSION THREAD

**Discussion by pecbeck** [mealie-recipes/mealie#3574] [General]
Is it possible to print the meal planner, other than screenshot?

**Comment by stuartw0** [comment]
I have this question also. Or export to pretty PDF.

**Comment by FreeMinded** [comment]
A simple & nice single page pdf view would very much help to get my wife motivated to use Mealie

**Comment by olskar** [comment]
This would be nice indeed

**Comment by stusemple** [comment]
I would value this. It would be nice to stick the plan for the week in the fridge
